volmdlr/faces.py

```diff
@@ -3540,7 +3535,8 @@ def merge_with(self, other_bspline_surface3d):
 
         if bspline_face3d.outer_contour3d.is_sharing_primitives_with(other_bspline_face3d.outer_contour3d):
 
-            xmin, xmax, ymin, ymax = self.xy_limits(other_bspline_surface3d)
+            # xmin, xmax, ymin, ymax = self.xy_limits(other_bspline_surface3d)
+            pass
```

Comment:
This if statement seems to have no effect, maybe we should just delete it?

Reply:
Comments have been deleted !

Reply:
what about the if statement?

Reply:
It's done!